src/mpc/mpc_common.cpp

```diff
@@ -57,3 +67,20 @@ void subcommand::usage(const po::options_description &options)
     subcommand_usage();
     std::cout << options << std::endl;
 }
+
+void populate_protoboard(libsnark::protoboard<FieldT> &pb, bool simple_circuit)
+{
+    if (simple_circuit) {
```

Comment:
It's kind of a debug / test thing  It is really valuable to have (it allows us to run a test script to execute the full pipeline on the build server and ensure all the command params work correctly and that the commands all read and write the correct file types).  But it's not particularly nice to have the flag there forever.

So I'm not sure what the ultimate solution should be.  Perhaps the most flexible solution would be to eventually support loading any circuit from a file?

Reply:
@AntoineRondelet (further to our discussion), libsnark appears to support `r1cs_constraint_system` streaming operators, so I think we could use that to implement some kind of `--load-?` flag.  (As discussed, this would fall back to using the full zeth circuit).

If you agree, for now, I might rename this to `--test-circuit` for the purposes of this PR, and then consider impelmenting the `--load` functionality later as a separate PR.

Reply:
I've implemented your suggestion of one executable `mpc` for joinsplit, and another executable for the test circuit.